src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```
@@ -1497,16 +1498,31 @@ Copyright (c) .NET Foundation. All rights reserved.
     </PropertyGroup>
   </Target>
 
+  <!--
+    ============================================================
+    _GetRestoreSourcesAsItems
+    Read $(RestoreSources) from the project as items.
+    Projects that do not have $(RestoreSources) will noop.
+    ============================================================
+  -->
+  <Target Name="_GetRestoreSourcesAsItems"
+    Returns="@(_RestoreSourcesItems)">
+    <ItemGroup Condition=" '$(RestoreSources)' != ''">
+      <_RestoreSourcesItems Include="$(RestoreSources.Split(';'))" />
```

```suggestion
      <_RestoreSources Include="$(RestoreSources.Split(';'))" />
```

I am not an MSBuild expert but looking at other targets and property names, I think adding `Items` keyword as suffix is not required. The usage of `@` symbol makes it clear that we are referring to Items. Please let me know if I am missing anything here.

---

We already have this section in the file.

```xml
 <!--
   ============================================================
   _GetRestoreTargetFrameworksAsItems
   Read $(TargetFrameworks) from the project as items.
   Projects that do not have $(TargetFrameworks) will noop.
   If $(TargetFramework) is specified globally, it'll be preferred over $(TargetFrameworks)
   ============================================================
 -->
 <Target Name="_GetRestoreTargetFrameworksAsItems"
   DependsOnTargets="_GetRestoreProjectStyle;_GetRestoreTargetFrameworkOverride"
   Returns="@(_RestoreTargetFrameworkItems)">
   <ItemGroup Condition=" '$(TargetFrameworks)' != '' AND '$(_TargetFrameworkOverride)' == '' ">
     <_RestoreTargetFrameworkItems Include="$(TargetFrameworks.Split(';'))" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFrameworks)' != '' AND '$(_TargetFrameworkOverride)' != '' ">
     <_RestoreTargetFrameworkItems Include="$(_TargetFrameworkOverride)" />
   </ItemGroup>
 </Target>
 ```
 
 For the sake of uniformity I added the the 'AsItems' suffix.